### DISCUSSION TITLE
Yarn out-of-memory error

### DISCUSSION THREAD

**Discussion by avdi** [mastodon/mastodon#22066] [Q&A]
I tried to spin up an instance on a Linode, but ran into an issue with Yarn asset precompilation failing with an out-of-memory error. Are there "minimum server requirements" documented somewhere?

**Comment by vikks** [comment]
Precompiling assets in the server takes up a whole lot of memory, but if you can compile assets locally and then push it to the servers, then mastodon runs in surprisingly low memory.
